.github/workflows/CI.yml

```diff
@@ -37,7 +41,50 @@ jobs:
             ${cmake_opts[@]}
           cmake --build build --parallel 2
           cmake --install build
-      - name: Test 3DThesis
+      - name: Test 3DThesis Snapshots
+        if: steps.build.outcome == 'success'
         run: |
-          $HOME/3dthesis/bin/3DThesis
-          mpirun -n 2 --oversubscribe $HOME/3dthesis/bin/3DThesis
+          cd /__w/3DThesis/3DThesis/tests/test_snapshot
```

Comment:
Not familiar with this folder - can we rely on it?

Reply:
Probably not reliable. Changed to $GITHUB_WORKSPACE, which is where the repo is cloned to, per the checkout action documentation, so that should be more reliable.